cmd/minikube/cmd/start.go

```diff
@@ -174,7 +173,7 @@ func init() {
 	provision.SetDetector(&provisioner.MinishiftProvisionerDetector{Delegate: provision.StandardDetector{}})
 }
 
-// initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
+// initStartFlags creates the CLI flags which needs to be passed on to 'minishift start'
```

Comment:
hmm, I meant 'cluster up' in this context. We are in 'minishift start', nothing to be passed on here, right?